server/layout/html.js

```diff
@@ -10,7 +10,7 @@ const render = ({
     css = [],
     js = [],
     react = 'main', // eslint-disable-line no-unused-vars
-    title = 'My React Application',
+    title = 'DECS MUI',
```

Comment:
On QA at the moment the title is "Correspondence System Management". Is this title actually used anywhere?

([Ideally](https://design.homeoffice.gov.uk/accessibility/layout-typography) these would be unique titles for every page, but that's for another time.)

Reply:
I'm not actually sure where/if it's used, I just saw it and figured 'My React Application' is a bit too generic